### PR TITLE
Update reliable-maintainable.md for Swan Lake GA

### PR DESCRIPTION
**Pull request by gimantha**
$Title

### CODE REVIEW

File: learn/why-ballerina/reliable-maintainable.md [hunk 1]
```diff
@@ -93,26 +95,30 @@ This `is` operator-based error checking pattern is very common and you would end
...
+import ballerina/tcp;
+
 function whois(string domain, tcp:Client whoisClient) returns string|error {
-   // If `writeBytes` failed with an error, then `check` makes
-   //  the function return that error immediately
-   check whoisClient->writeBytes(domain.toBytes());
-   byte[] bytes = check whoisClient->readBytes();
-   return string:fromBytes(bytes);
+    // If `writeBytes` failed with an error, then `check` makes
+    //  the function return that error immediately
```
Comment (praneesha) [inline learn/why-ballerina/reliable-maintainable.md:102]:
```suggestion
    //  the function to return that error immediately.
```

File: learn/why-ballerina/reliable-maintainable.md [hunk 2]
```diff
@@ -149,20 +155,20 @@ Abnormal errors can be reported using the `panic` statement. Some language const
 
 ```ballerina
 function toInt(any a) returns int {
-   // This is a programming bug
-   // Raise a panic if the value of `a` is not an `int`
-   return <int>a;
+    // This is a programming bug
```
Comment (praneesha) [inline learn/why-ballerina/reliable-maintainable.md:158]:
```suggestion
    // This is a programming bug.
```

File: learn/why-ballerina/reliable-maintainable.md [hunk 3]
```diff
@@ -149,20 +155,20 @@ Abnormal errors can be reported using the `panic` statement. Some language const
...
+    // Raise a panic if the value of `a` is not an `int`
```
Comment (praneesha) [inline learn/why-ballerina/reliable-maintainable.md:159]:
```suggestion
    // Raise a panic if the value of `a` is not an `int`.
```

**Review by praneesha — APPROVED** [review]

File: learn/why-ballerina/reliable-maintainable.md [hunk 4]
```diff
@@ -93,26 +95,30 @@ This `is` operator-based error checking pattern is very common and you would end
...
+import ballerina/tcp;
+
 function whois(string domain, tcp:Client whoisClient) returns string|error {
-   // If `writeBytes` failed with an error, then `check` makes
-   //  the function return that error immediately
-   check whoisClient->writeBytes(domain.toBytes());
-   byte[] bytes = check whoisClient->readBytes();
-   return string:fromBytes(bytes);
+    // If `writeBytes` failed with an error, then `check` makes
+    //  the function to return that error immediately.
```
Comment (MaryamZi) [inline learn/why-ballerina/reliable-maintainable.md:102]: Why do we need `to` here?

It's like

"make someone do something" vs "make someone to do something". @praneesha

Comment (praneesha) [reply learn/why-ballerina/reliable-maintainable.md:102]: Sorry, my bad. +1 to remove `to`. @gimantha - Can you send the fix for this?

**Pull request merged**